src/test/rustdoc-json/primitive.rs

```diff
@@ -0,0 +1,9 @@
+// edition:2018
+
+#![feature(doc_primitive)]
+#[doc(primitive = "usize")]
+mod usize {}
```

Comment:
Can you also test the behavior when the primitive is only defined in libstd, not the current crate?

Reply:
Of course

Reply:
Added tests to check `crate_id` of methods of `usize` does not match to local crate's. We cannot check if they come from libstd because the `std` module item is not in the index, but hope it suffices.

Reply:
> Added tests to check crate_id of methods of usize does not match to local crate's. 

In that case, this `mod usize` is doing nothing and should be removed.

Reply:
`doc(primitive)` requires an item to be defined right after itself, so we need this meaningless `mod usize`.

Reply:
No, my point is that `doc(primitive)` is doing nothing. If it's using the methods in libstd then it doesn't matter where the primitive is defined.

Reply:
Actually, I'm not sure this is the right approach at all - the original problem was that documenting libcore wouldn't show the primitive impls, right? but we don't want to show them on *all* crates unconditionally, only crates where the primitive is defined.